content/en/docs/contribute/style/content-guide.md

```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
+            1. Example: including [kubadm](https://github.com/kubernetes/kubeadm) installation and troubleshooting instructions because the kubeadm project does not have its own documentation in github or on a separate website   
+    1. Not Allowed:
+        1. Adding content that duplicates documentation in another repository
+            1. Examples: 
+                1. Including minikube installation and configuration instructions; minikube has its own [documentation](https://minikube.sigs.k8s.io/docs/) that provides those instructions
+                1. Including instructions for installing Docker, CRI-O, containerd, and other container runtimes on various operating systems
+                1. Including instructions for installing Kubernetes on production environments using various projects:
+                    1. Kubernetes Rebar Integrated Bootstrap (KRIB) is a vendor-specific project and content belongs in the vendor's documentation
+                    1. [Kubernetes Operations (kops)](https://github.com/kubernetes/kops) has installation instructions and tutorials in its GitHub repository
+                    1. [Kubespray](https://kubespray.io) has its own documenation  
+        1. Adding content that explains how to perform a task using a vendor-specific product or an open source project that is not a CNCF project or a project in the kubernetes or kubnetes-sigs GitHub organizations
+        1. Adding content on how to use a CNCF project or a project in the kubernetes or kubnetes-sigs GitHub organizations if the project has its own documentation
 1. Detailed technical content about how to use a non-Kubernetes project or how that project is designed
 
     Adding this type of content to the Kubernetes documentation is not allowed.     
 1. Content that describes a non-Kubernetes project
-    - Allowed:    
-        - Adding a brief introductory paragraph about a CNCF project or a project in the kubernetes or kubernetes-sigs GitHub organizations; the paragraph may contain links to the project        
-    - Not Allowed:    
-        - Adding content describing a vendor-specific product
-        - Adding content describing an open source project that is not a CNCF project or a project in the kubernetes or kubnetes-sigs GitHub organizations
-        - Adding content that duplicates documentation from another project, regardless of source repository
-            - Example: adding [Kubernetes in Docker (KinD)](https://kind.sigs.k8s.io) documentation to the Kubernetes documentation
+    1. Allowed:    
+        1. Adding a brief introductory paragraph about a CNCF project or a project in the kubernetes or kubernetes-sigs GitHub organizations; the paragraph may contain links to the project        
+    1. Not Allowed:    
+        1. Adding content describing a vendor-specific product
+        1. Adding content describing an open source project that is not a CNCF project or a project in the kubernetes or kubnetes-sigs GitHub organizations
```

Comment:
nit: line 87, spelling kubnetes-sigs
The list of accepted projects (CNCF, kubernetes, kubernetes-sigs Github organizations) is a lot to parse. Is there any way to make a key or acronym?

Reply:
This will be done in a separate PR.  I've reverted my content-guide.md changes.